ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po

```diff
@@ -0,0 +1,492 @@
+msgstr "Tem certeza de que deseja excluir esse comentário?"
+
+#: ckanext/datarequests/templates/datarequests/snippets/comment_item.html:29
+msgid "commented"
+msgstr "comentou"
+
+#: ckanext/datarequests/templates/datarequests/snippets/comments.html:3
+msgid "Current Discussion"
+msgstr "Discussão atual"
+
+#: ckanext/datarequests/templates/datarequests/snippets/comments.html:13
+msgid "This data request has not been commented yet"
+msgstr "Esta solicitação de dados ainda não foi comentada"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_form.html:17
+msgid "eg. Data Request Name"
+msgstr "Ex.: Nome da solicitação de dados"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_form.html:21
+msgid "eg. Data Request description"
+msgstr "Ex.: Descrição da Solicitação de dados"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_form.html:29
+msgid "No organization"
+msgstr "Nenhuma organização"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_form.html:43
+msgid "Are you sure you want to delete this data request?"
+msgstr "Tem certeza de que deseja excluir este pedido de dados?"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_form.html:44
+msgid "Delete"
+msgstr "Excluir"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_list.html:1
+msgid "No Data Requests found"
+msgstr "Nenhuma solicitação de dados encontrada"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_list.html:14
+msgid "No Data Requests found with the given criteria"
+msgstr "Nenhuma solicitação de dados encontrada com os critérios informados"
+
+#: ckanext/datarequests/templates/datarequests/snippets/datarequest_list.html:16
+msgid "How about creating one?"
+msgstr "Que tal criar um?"
+
+#: ckanext/datarequests/templates/datarequests/snippets/edit_datarequest_form.html:4
+msgid "Update Data Request"
+msgstr "Atualizar Solicitação de Dados"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:5
+msgid "{0} statistics"
+msgstr "{0} estatísticas"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:10
+msgid "dataset"
+msgstr "conjunto de dados"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:10
+msgid "datasets"
+msgstr "conjuntos de dados"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:16
+msgid "organization"
+msgstr "organização"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:16
+msgid "organizations"
+msgstr "organizações"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:22
+msgid "group"
+msgstr "grupo"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:22
+msgid "groups"
+msgstr "grupos"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:28
+msgid "related item"
+msgstr "item relacionado"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:28
+msgid "related items"
+msgstr "itens relacionados"
```

Comment:
itens or items?

Reply:
That's correct. In portuguese the plural of "item" is "itens"...